posts/bayesian-model-comparison/index.qmd

```diff
@@ -308,30 +316,31 @@ def get_t_dist_posterior_probs(
   # * Otherwise compute the probability for the specified t-distribution.
```

Comment:
here we should have `prob_centre = 0.0`

Reply:
cioe?

Reply:
if we hit this block the return will fail as we won't have `prob_centre`. Given that we hit that when var is zero it means that we can state the full prob is where the mean is. In fact this means we would need `prob_centre = float(rope[0] <= mean <= rope[1])`

Reply:
I lie. You should do what we do below, with 1-. So if you pull the `prob_centre` outside of else block we're good